data/gnome-shell-theme.gresource.xml

```diff
@@ -41,6 +41,12 @@
     <file>endless-auto-updates-scheduled-symbolic.svg</file>
```

Comment:
I don't think this commit should be authored to Mario. You probably can use Sam, or myself as the original author.